Console/Command/QueueShell.php

```diff
@@ -291,7 +413,7 @@ protected function _listTasks() {
 		}
 	}
 	
-	function out($str='') {
+	function out($str = null, $newlines = 1, $level = Shell::NORMAL) {
```

Comment:
Probably?  I just added it because strict mode was complaining the method signature didn't match the one it was overriding.

I'll update it.